ports/geckolib/glue.rs

```diff
@@ -343,6 +343,19 @@ pub extern "C" fn Servo_GetComputedValuesForPseudoElement(parent_style: *mut Ser
 }
 
 #[no_mangle]
+pub extern "C" fn Servo_GetComputedValuesForOtherNonElement(parent_style: *mut ServoComputedValues)
```

Comment:
I think it would be nice to make this more general and call it Servo_InheritComputedValues, to make it clear that this just resets the non-inherited structs.